packages/cli/src/ceramic-daemon.ts

```diff
@@ -489,6 +495,31 @@ export class CeramicDaemon {
     })
   }
 
+  async getModelsFromIndex(req: Request, res: Response): Promise<void> {
+    const indexedModelStreams = await this.ceramic.admin.getModelsFromIndex()
+    res.json({
+      models: indexedModelStreams.map(modelStream => StreamUtils.serializeState(modelStream.state))
```

Comment:
+1, @stbrody . But also happy to add the `is_indexed` field anytime, if we feel it's useful, @v-stickykeys , @willex . Please share your thoughts :).

Reply:
`is_indexed` is meant to provide a logical delete option & filter for actively indexed models on the node via the `ceramic_models` table. So, looking for actively indexed models against the DB we should filter with `is_indexed IS TRUE`.

Reply:
right, I get why we have this field in the database.  My question is why we would expose it in the HTTP api to our end users, vs just filtering out the models where is_indexed is false in the db

Reply:
For context, see the spec for GET requests: https://www.notion.so/threebox/Admin-API-for-dynamic-indexing-039ea68c9d374c1987b3e09cd745296a#2a22fa99a01844778045595d1c26c881

Reply:
- response body contains models and their fields from db
    - `{models: [{model, is_indexed}, ...]}`

Reply:
ah IC, I had a discussion around that with @pawartur earlier.

Since we allow explicit deletes (and potentially future implicit deletes through deployments as discussed with the `PUT` method) via the API it is a way to confirm that a previously indexed model was indeed deactivated without the need to directly connect to the DB. 

If we only show active ones by default we lose an important part of the audit trail that e.g. a model was deactivated by accident by a different developer or admin and thats why it doesn't show up etc. 

There is also the option that we drop `is_indexed` and just implement a filter flag on the `GET` endpoint to query for deactivated models as well.

Reply:
I'm not really convinced.  To me it's a pretty simple API to just see which models are currently indexed and nothing else.  Making apps inspect and filter on the is_indexed field seems to complicate the API for little value.

That said I don't feel *that* strongly about it.  If yall are all in agreement to include the field then I won't stand in the way